use paths for rgb that make more sense in Linux

http://bugs.gentoo.org/139423

--- netpbm-10.34/pm_config.in.h
+++ netpbm-10.34/pm_config.in.h
@@ -67,8 +67,8 @@
 #define RGB_DB2 "PBMplus_Dir:RGB.TXT"
 #define RGB_DB3 "PBMplus_Dir:RGB.TXT"
 #else
-#define RGB_DB1 "/usr/lib/X11/rgb.txt"
+#define RGB_DB1 "/usr/share/X11/rgb.txt"
-#define RGB_DB2 "/usr/openwin/lib/rgb.txt"
+#define RGB_DB2 "/usr/lib/X11/rgb.txt"
 #define RGB_DB3 "/usr/X11R6/lib/X11/rgb.txt"
 #endif
 
